packages/gitbook/src/components/AdminToolbar/Toolbar.module.css

```diff
@@ -0,0 +1,40 @@
+.arcMenu {
+    --arc-width: 505px;
+    --arc-height: 400px;
+    --arc-radius: 34%;
+    --start-distance: -240px;
+    --spread-distance: 45px;
+}
```

Comment:
This controls the arc menu that pops open when the button for hiding options is clicked. 

We use a border-radius on an invisible parent combined with offset-distance on each child button to then have them 'follow' [in this file](https://github.com/GitbookIO/gitbook/pull/3698/files#diff-06c3e3656f455c6c47ff2f4a59e118ca5b7d4d08f30de6e7dbac739ce03e54b4). 

Better visualized with a dashed border below:

<img width="1574" height="680" alt="CleanShot 2025-10-03 at 12 17 50@2x" src="https://github.com/user-attachments/assets/39d15671-4fc0-47d2-ad55-e050c0e6c750" />